src/services/sync/strategies/TSP/TSP.js

```diff
@@ -0,0 +1,163 @@
+const rp = require('request-promise-native');
+const url = require('url');
+const moment = require('moment');
+const { JWE, JWK, JWS } = require('jose');
+const uuid = require('uuid/v4');
+const commons = require('@schul-cloud/commons');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module '@schul-cloud/commons'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4605061)